service/controller/app/resource/status/create.go

```diff
@@ -70,12 +70,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		desiredStatus = v1alpha1.AppStatus{
 			AppVersion: chartStatus.AppVersion,
 			Release: v1alpha1.AppStatusRelease{
-				LastDeployed: chartStatus.Release.LastDeployed,
-				Reason:       chartStatus.Reason,
-				Status:       chartStatus.Release.Status,
+				Reason: chartStatus.Reason,
+				Status: chartStatus.Release.Status,
 			},
 			Version: chartStatus.Version,
 		}
+		if chartStatus.Release.LastDeployed != nil {
+			desiredStatus.Release.LastDeployed = *chartStatus.Release.LastDeployed
+		}
```

Comment:
LastDeployed because an optional field so we need to handle nils or the operator crashes.